exchanges/kraken/kraken_types.go

```diff
@@ -409,6 +409,7 @@ type AddOrderOptions struct {
 	ClosePrice     float64
 	ClosePrice2    float64
 	Validate       bool
+	TimeInForce    string
```

Comment:
Here as well. Either remove type or force usage of type.

Reply:
done